.husky/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+CI=true npm run ci
+npm run test:e2e
```

Comment:
Is this like an `if` statement? It runs the the first command if CI is `true` or the second command if it's `false`? This syntax is a bit mind-bending if it's intended.

Reply:
No, these are two seperate commands, like you would run in your shell. if CI is false the test command is in watch mode automatically i think, thats why i put that there

Reply:
Oh, got it. `npm run ci` is exactly the same as `npm run test`. Thanks.